source/docs/articles/local/drupal-drush-command-line-utility.md

```diff
@@ -8,39 +8,20 @@ keywords: Drupal drush, command line, drupal, terminus drush, cli
+Drush-savvy developers should also install and utilize [Terminus](/docs/articles/local/cli/), a command-line interface that allows you to control your Pantheon account and sites. Virtually anything you can do in the Dashboard, you can script with Terminus. It can also make remote Drush calls on your environments without having Drush installed locally, eliminating incompatibility issues between locally and remotely installed versions of Drush.
 
-Using Terminus to operate Drush commands on your site environments negates the issues below, which stem from incompatibilities between locally and remotely installed versions of Drush. All of the commands below can be run from Terminus instead of using Drush aliases. For more information, see our guide on [Managing Drupal Sites with Terminus and Drush](/docs/guides/terminus-drupal-site-management/).
+You can run all of the commands below from Terminus instead of using Drush aliases. For more information, see our guide on [Managing Drupal Sites with Terminus and Drush](/docs/guides/terminus-drupal-site-management/).
 
 ## Drush Versions
-Pantheon currently has Drush version 5.10.1 installed. You can run Drush 5.x, 7.x, and 8.x on your local installation to interact with your Pantheon Drupal installations. Pantheon alias files are not compatible with 6.x, however you can execute commands locally with this version by including the `--strict=0` option.
+For details on managing remote and local Drush versions, see [Managing Drush Versions on Pantheon](/docs/articles/local/drush-versions).
 
-For upgrade information, see [Introducing Drush 8](https://pantheon.io/blog/introducing-drush-8).
-
-#### Drush 7
-Create a new policy file that changes all remote aliases to use Drush 7 instead of the default version of Drush, but only if the target is the Pantheon platform. Our `hook_drush_sitealias_alter` function looks like this:
-```php
-function policy_drush_sitealias_alter(&$alias_record) {
-  // Fix pantheon aliases!
-  if ( isset($alias_record['remote-host']) &&
-      (substr($alias_record['remote-host'],0,10) == 'appserver.') ) {
-    $alias_record['path-aliases']['%drush-script'] = 'drush7';
-  }
-}
-```
-
-With this policy file in place, you are able to use the latest version of Drush on Pantheon:
-```
-$ drush @pantheon.my-great-site.dev version
-Drush Version   :  7.0.0-rc1
-```
-
-For more details, see our [Fix Up Drush Site Aliases with a Policy File](https://pantheon.io/blog/fix-drush-site-aliases-policy-file) blog post.
 ## Install Drush Aliases Locally
-Adding Pantheon aliases to your local Drush aliases file will allow you to run Drush calls against your Pantheon site environments. There are two methods for obtaining the aliases:
+Adding Pantheon aliases to your local Drush aliases file will allow you to run Drush calls against your Pantheon site environments. You don't need to download the Drush aliases file if you're using Terminus to invoke Drush.

```

Comment:
"Downloading the Pantheon aliases to your local Drush configuration directory" would be better here. We do not want folks to take this literally and modify their aliases.drushrc.php file. This updated description matches the process that follows.